src/common/utils.cpp

```diff
@@ -71,6 +85,34 @@ std::tuple<double, double> leg_eval(int n, double x) {
   return {p2, n * (x * p2 - p1) / (x * x - 1)};
 }
 
+double cyl_bessel_i(double nu, double x) noexcept {
+#if defined(FINUFFT_HAVE_STD_CYL_BESSEL_I)
+  return std::cyl_bessel_i(nu, x);
+#else
```

Comment:
Is this a local implementation in case not in C++ stdlib? please add comment

Reply:
Mac does not provide cyl_bessel_i. The apple clang implementation is not standard compliant.